Glitch/dist/metadata.json

```diff
@@ -18,5 +18,5 @@
     "community"
     ],
     "category": "other",
-    "reqExp": "(([a-z0-9]+)[.])?glitch."
+    "reqExp": "(([a-z0-9]+)[.])?glitch[.]([a-z0-9]+)"
```

Comment:
you typed reQexp instead of reGexp so the right is: regExp

Reply:
I have no idea why I didn't catch that.